apps/demos/Demos/Chat/AIIntagration/jQuery/index.js

```diff
@@ -0,0 +1,189 @@
+const deployment = 'gpt-4o-mini';
+const apiVersion = '2024-02-01';
+const endpoint = 'https://public-api.devexpress.com/demo-openai';
+const apiKey = 'DEMO';
+
+const store = [];
+let messages = [];
+let lastAIResponse = null;
+
+const user = {
+  id: 'user',
+};
+
+const assistant = {
+  id: 'assistant',
+  name: 'Virtual Assistant',
+};
```

Comment:
let's move users and constants to a separate file `data.js`?

Reply:
[80c47e37d89c79163a8463520cfe9ddef1e3beef]